spark/src/test/scala/org/apache/comet/CometArrayExpressionSuite.scala

```diff
@@ -232,6 +232,21 @@ class CometArrayExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelp
     }
   }
 
+  test("array_max") {
+    withSQLConf(CometConf.COMET_EXPR_ALLOW_INCOMPATIBLE.key -> "true") {
```

Comment:
Can we document why we need this? Is there a specific scenario where Comet's array_max behavior differs from Spark?